drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -621,6 +621,35 @@ def ce_referrals(**args)
       resolve_ce_referrals(Hmis::Ce::Referral.all, **args)
     end
 
+    field :table_config_lookup, Types::TableConfigLookup, null: false
+    def table_config_lookup
+      {}
+    end
```

Comment:
discussed using [resolver](https://graphql-ruby.org/fields/resolvers.html) as another possible pattern to explore. (similar to Mutation which is a subclass of resolver)

```
    field :table_config_lookup, resolver: Types::TableConfigResolver
```